src/game/gameOverModal.js

```diff
@@ -0,0 +1,24 @@
+import styles from "./gameOverModal.scss";
+
+const modal = document.createElement("div");
+modal.classList.add("modal");
+
+export function displayResult(GAME_RESULT, GAME_WIN_OR_LOSE, POINTS) {
+  modal.innerHTML = `<div class="modal__content">
+                        <h2>${GAME_RESULT}!</h2>
+                        <h3>You ${GAME_WIN_OR_LOSE} with ${POINTS} points</h3>
+                        <div class="modal__actions">
+                            <a href="#" class="modal__new-game-js">Start New Game</a>
+                            <a href="/highscores.html">High Scores</a>
+                            <a href="/main.html">Return</a>
+                        </div>                        
+                    </div>`;
+  document.querySelector("body").prepend(modal);
+  document
+    .querySelector(".modal__new-game-js")
```

Comment:
`modal__new-game-js` what is this for?

Reply:
it's for "Start New Game" button. It will allow us to close modal and start new game
More logic than closing modal will be added when we will have function for stating game

Reply:
Ahh. Just forgot that on this modal was a "Start New Game" 😄